packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx

```diff
@@ -63,6 +63,8 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
   private _paletteRef: HTMLDivElement | null = null
   private _mouseMoveListener?: { remove(): void }
   private _mouseUpListener?: { remove(): void }
+  //@ts-expect-error Property 'window' does not exist on type 'ColorPalette'
+  private _paletteOffset = (getFontSize(this.window) / 16) * 2
```

Comment:
I would use the `px()` util, then you don't need the ts-expect-error. This util gives back the px value of '0.125rem', depending on the current document font size.

There is another problem: this value is still hardcoded. As far as I understand, this value comes from somewhere, (I assume it is "2 times the indicator border width", but you are more familiar with this). These are usually theme variables, meaning they change if someone overrides them. Try to make it dependent on the theme variable connected to this logic. Let me know if you need help with it. 

```suggestion
  private _paletteOffset = px('0.125rem')
```